spec/periodicChecksSpec.js

```diff
@@ -129,15 +129,15 @@ describe('Periodic Checks Module', () => {
   };
 
   beforeEach(async () => {
-    spyOn(scheduler, 'createScheduler').and.callFake(() => {});
+    spyOn(scheduler, 'createScheduler').and.callFake(() => { });
 
     github = {
       issues: {
         createComment: jasmine
```

Comment:
```suggestion
        createComment: jasmine.createSpy('createComment').and.callFake(() => {})
```

Reply:
this will give max-len issue

Reply:
What is the length limit, since this line is 80 chars and I though we allow max 80 chars (<=80)

Reply:
we allow max 80, and this line is going over.